app/controllers/cori_questions_controller.rb

```diff
@@ -1,11 +1,17 @@
 class CoriQuestionsController < ApplicationController
   before_action :authenticate_user!
-  
+
   def index
     if params[:yes_or_no] == "no"
-      redirect_to "/pages/nocori"
+        redirect_to "/pages/nocori"
     elsif params[:yes_or_no] == "yes"
-      redirect_to new_sex_offender_path
+      respond_to do |format|
+        format.html {
+        redirect_to new_sex_offender_path }
+        format.json { render json: { action: "/sex_offenders", id: "sex-offender-question",
```

Comment:
Line is too long. [91/80]<br>Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.